### ISSUE TITLE
Tweaks to widget copy

### ISSUE THREAD

**Issue by noisecapella** [mitodl/open-discussions#1656]
From https://github.com/mitodl/open-discussions/pull/1646#issuecomment-452152456

Can we change the copy a bit? I realize this may require customizing some common elements. If it's out of scope for this PR, we can open a separate issue.

    Title: “Add text widget” (instead of "Edit widget")
    Box 1: “Title” (instead of "Widget title")
    Box 2: “Body” (instead of "Source")

Also, can we change the placeholder text to "Widget text" instead of "Enter widget text"

And change the button text from "Create widget" to "Add widget"

**Comment by noisecapella** [comment]
I think we will need to add a description to the field type as well as the widget type before we can do this. #1592

**Comment by rhysyngsun** [comment]
Adding to this, the label for the affirmative action button when editing a widget is currently labelled "Update Widget", which when testing a PR I absent mindedly thought meant would save my changes and then I refreshed the page, losing my changes in the process. We might want to change this label to something that wouldn't give someone that impression.

And/or possibly we may want to hook into `window.onbeforeunload` and prompt the user before a navigation occurs with unsaved changes.
